pipelines/common/src/main/java/com/google/fhir/analytics/DwhFiles.java

```diff
@@ -65,6 +65,8 @@ public class DwhFiles {
 
   public static final String TIMESTAMP_FILE_END = "timestamp_end.txt";
 
+  public static final String TIMESTAMP_FILE_TRANSACTION = "timestamp_transaction.txt";
```

Comment:
Please add a "bulk" word somewhere in both the constant name and the actual filename.